Easter Egg Hunt Checklist #1

swoocn opened the pull request:
Simple change to remove the Business Types field; hardcoded to 'General'.

Tested change via sandbox and verified nothing broke.
Attached is part of the result for the tested shop using find query against Mongo DB ->

![image](https://github.com/swoocn/map-of-pi-redux/assets/72060658/163eec9c-61ed-4473-8689-9387a2dcf2b3)

@soleil00 - looks like we still have a big payload with the images, btw.

swoocn committed:
Comment out business types field.

swoocn committed:
Assign static value of 'General' Business Type.

soleil00 commented:
okay i will change that, we are still sending base64 as image which i will replace with actual image upload

DarinHajou commented on frontend/src/app/business/business-settings/business-settings.component.html:
Hey team,

Great job on the PR! I noticed we're subscribing to 'onLangChange' in 'BusinessSettingsComponent'. Could we make sure we're also cleaning up those subscriptions to avoid any potential memory leaks? A quick unsubscribe in 'ngOnDestroy' should do the trick.

swoocn replied:
Great catch!  I will get it implemented.. or since the Business Types section has been removed I can just remove it as well.

swoocn replied:
Made some follow-up changes @DarinHajou.  Thx!

swoocn committed:
DH PR Comment; removed Business Types from component to eliminate potential memory leaks.

swoocn committed:
Following up DH PR Comment; implemented garbage collection for subscription in Map Component.

zolcsi commented on frontend/src/app/business/business-settings/business-settings.component.spec.ts:
```diff
@@ -1,5 +1,7 @@
+
 // import { BusinessSettingsComponent } from './business-settings.component';
 
 // describe('BusinessSettingsComponent', () => {
```
this testcase seems to be obsolete, let's delete it

zolcsi commented on frontend/src/app/business/business-settings/business-settings.component.html:
```diff
@@ -23,7 +23,7 @@
           ">{{ 'BUSINESS.REGISTRATION.VALIDATIONS.BUSINESS_DESCRIPTION' | translate }}</span>
     </div>
-    <div class="business-registration__field business-registration__field-middle">
+    <!-- <div class="business-registration__field business-registration__field-middle">
```
if we don't need a code, let's delete it, we still have git to find it if we were in need

zolcsi reviewed (commented):
Let's remove dead code, we see clearer then

swoocn committed:
Merge branch 'dev' into easter-egg-hunt-list-1

swoocn committed:
ZM PR Comment; remove dead code suggestion.

zolcsi commented on frontend/src/app/home/map/map.component.ts:
```diff
@@ -43,21 +46,22 @@ export class MapComponent implements OnInit {
-    });
+    private logger: NGXLogger) {
+      this.options = this.geolocationService.getMapOptions();
+      this.geolocationService.geolocationTriggerEvent$.subscribe(() => {
```
this subscription should be also cleaned up

swoocn replied:
This has been garbage collected; good catch!

zolcsi reviewed (approved):
LGTM

swoocn committed:
ZM PR Comment; garbage collect geolocationSubscription.
Update versioning; remove styling instance to resolve 'Failed to inline external stylesheet' error.
